external/dhcpc/dhcpc.c

```diff
@@ -332,7 +335,14 @@ static uint8_t dhcpc_parseoptions(struct dhcpc_state *presult, uint8_t *optptr,
 		case DHCP_OPTION_DNS_SERVER:
 			/* Get the DNS server address in network order */
 
-			memcpy(&presult->dnsaddr.s_addr, optptr + 2, 4);
+			total_dns = (*(optptr+1))/4;
+			total_dns = (total_dns > 2) ? 2 : total_dns;
+			uint8_t *start_addr = optptr+2;
+			for(int i = 0;i < total_dns;++i){
```

Comment:
please add space and follow TizenRT coding guidelines

Reply:
Done